kolibri/core/assets/src/views/app-body.vue

```diff
@@ -10,9 +10,17 @@
       type="indeterminate"
       :delay="false"
     />
-    <div v-else class="wrapper">
-      <error-box v-if="error" />
-      <slot></slot>
+
+    <div
+      v-else
+      :style="bodyPadding"
+      class="body-wrapper"
+    >
+      <app-error v-if="error" />
+
+      <slot v-else>
```

Comment:
Same line

Reply:
Can we wait till this PR goes into develop to get this granular?